src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/subheadline/index.js

```diff
@@ -0,0 +1,14 @@
+import convertParagraph from '../paragraph';
```

Comment:
Is it not best practice to name the default export the same as the file name?

Reply:
Interesting, ive never see anyone suggest either way before, maybe? :P